.tx/config

```diff
@@ -1,7 +1,7 @@
 [main]
 host = https://www.transifex.com
 
-[o:openmrs:p:openmrs3:r:openmrs-core]
+[o:openmrs:p:openmrs:r:openmrs-core]
```

Comment:
This is correct, right @ibacher?       

Reply:
Yes. Core isn't part of the openmrs3 project. It's part of the older "openmrs" project.